packages/instrumentation-sequelize/package.json

```diff
@@ -54,6 +54,7 @@
         "opentelemetry-instrumentation-testing-utils": "^0.5.0",
         "pg": "^8.4.2",
         "sequelize": "^5.22.0",
+        "sqlite3": "^5.0.2",
```

Comment:
@nirsky what is the usage for `sqlite3`?

Reply:
tests